### ISSUE TITLE
Allow ngx-build-plus:externals without ngx-build-plus:wc-polyfill

### ISSUE THREAD

**Issue by kylecordes** [manfredsteyer/ngx-build-plus#159]
Currently, if you use `ngx-build-plus:externals` without using  `ngx-build-plus:wc-polyfill` first, it fails with an error.

Fix/feature request:

The externals feature is useful for any project that wants to externalize libraries, including projects that don't use web components at all. Make the externals schematic not require anything WC-related to happen first. (Though it might need to adjust for external non-WC polyfills.)

**Comment by manfredsteyer** [comment]
Good point. I think, it works, however, by default it generates this line in ``angular.json``:

```
 "scripts": [
   [...]           
   "node_modules/@angular/elements/bundles/elements.umd.js",
   [...]           
            ]
```

Hence, we need to remove this line or install angular/elements to make this work.

Is this what you are referring to?

Best Wishes,
Manfred
